crowdfunding_compassion/templates/project_page.xml

```diff
@@ -87,22 +87,24 @@
                                     <t t-esc="project.description"/>
                                 </p>
                                 <t t-if="project.presentation_video">
-                                    <iframe class="my-3" width="100%" height="400px" t-att-src="project.presentation_video" frameborder="0"/>
+                                    <iframe class="my-3" width="100%" height="400px" t-attf-src="#{project.presentation_video}" frameborder="0"/>
```

Comment:
This would have also worked with previous version. t-att takes variable and  interpret directly as python code. t-attf writes as a string and uses special notation #{} to inject python. So the previous version was even better in my opinion.

Reply:
ohh ok I didn't know, I will change it back